CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 - The `go.opentelemetry.io/contrib/config` add support to configure periodic reader interval and timeout. (#5661)
-
+- The codespell workflow has been added to GitHub Action, and also populate related makefile command and requirements.txt file. (#5683)
```

Comment:
This change is internal to the repository (it has no impact on the provided libraries). So there's no changelog needed.

Reply:
Yup, thanks for pointing it out. I will get rid of this line mate.